lavalink/WebSocket.py

```diff
@@ -80,9 +83,10 @@ def connected(self):
         bool
             ``True`` if the reconnection attempt was successful.
         """
-        log.info('Connection closed; attempting to reconnect in 30 seconds')
+        self._node.set_offline()
+        log.info('Connection closed; attempting to reconnect in 10 seconds')
         for a in range(0, self._ws_retry):
-            await asyncio.sleep(30)
+            await asyncio.sleep(10)

```

Comment:
Might be worth considering an exponential backoff?

Reply:
Yes, but i think it should flatline somewhere

Reply:
Yeah, should start with a static backoff and then grow with each retry attempt after 2~3 failed attempts